src/frontend/src/optimizer/property/distribution.rs

```diff
@@ -40,8 +90,8 @@ impl Distribution {
                 Distribution::Single => DistributionMode::Single,
                 Distribution::Broadcast => DistributionMode::Broadcast,
                 Distribution::HashShard(_) => DistributionMode::Hash,
-                // TODO: Should panic if AnyShard or Any
-                _ => DistributionMode::Single,
+                // TODO: add round robin DistributionMode
```

Comment:
When is it possible to have round-robin? 🤣

Reply:
To increase parallelism of processing.

Reply:
Round robin will cause a lot of problems. For example, in two-phase agg, it will cause row_count to be negative. I think long time ago we've decided to use hash distribution of the values for the local phase.

Reply:
But it's okay to have it in batch, IMHO.

Reply:
Yes, it should be used carefully.